src/set.js

```diff
@@ -0,0 +1,72 @@
+export default class Set {
+  constructor( arrayOfElements ) {
+    let _items = arrayOfElements || []
+    let _size = arrayOfElements ? arrayOfElements.length : 0
```

Comment:
the ternary is a good idea but I think you should use the built in functions you made in order to determine size of your set.